medical-portal/src/API/RSBC.DMF.MedicalPortal.API/AutoMapper.cs

```diff
@@ -38,11 +38,23 @@ public MappingProfile()
                 .ForMember(dest => dest.CreatedOn, opt => opt.MapFrom(src => src.CreatedOn))
                 .ForMember(dest => dest.LoginId, opt => opt.MapFrom(src => src.Provider.Id));
 
+            CreateMap<Document, ViewModels.DmerDocument>()
+                .ForMember(dest => dest.DocumentId, opt => opt.MapFrom(src => src.DocumentId))
+                .ForMember(dest => dest.LoginId, opt => opt.MapFrom(src => src.Provider.Id))
+                .ForMember(dest => dest.IdCode, opt => opt.MapFrom(src => src.Case.CaseNumber))
+                .ForMember(dest => dest.FullName, opt => opt.MapFrom(src => src.Case.Person.FullName))
+                .ForMember(dest => dest.BirthDate, opt => opt.MapFrom(src => src.Case.Person.Birthday))
+                .ForMember(dest => dest.DueDate, opt => opt.MapFrom(src => src.ComplianceDate));
```

Comment:
I won't make you change this but ComplianceDate is a Dynamics term. DueDate is the more generic term we use outside of Dynamics. So ViewModels should be using the name "DueDate", not "ComplianceDate".